src/Sarif.Multitool.FunctionalTests/TestData/UrisMustBeValid/Invalid.sarif

```diff
@@ -5,7 +5,13 @@
     {
       "tool": {
         "name": "CodeScanner",
-        "downloadUri": "ht%tp://www.example.com/tools/codescanner/download.html"
+        "downloadUri": "ht%tp://www.example.com/tools/codescanner/download.html",
+        "rulesMetadata": [
```

Comment:
>rulesMetadata [](start = 9, length = 13)

as you recall, certain invalid test files _must_ be kept in conformance with current SARIF format because they can't be put through the prerelease transformer without having their essential flaws (upon which testing depends) cleaned up. :)

Reply:
Got it.

---
In reply to: [255249407](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255249407) [](ancestors = 255249407)